src/main/kotlin/org/jetbrains/research/transformations/Anonymization.kt

```diff
@@ -0,0 +1,75 @@
+@file:Suppress("UNCHECKED_CAST")
+
+package org.jetbrains.research.transformations
+
+import com.github.gumtreediff.tree.ITree
+import com.github.gumtreediff.tree.Tree
+import com.github.gumtreediff.tree.TreeContext
+
+object Anonymization: Transformation {
+    override val metadataKey = "anonymization"
+    private var anonymNamesMap: MutableMap<String, String> = mutableMapOf()
+    private var numOfVariables = 0
```

Comment:
Sorry, why do you have the global variables for metadata? I thought that we should create it each time when we are trying to apply the transformation. I mean it would be better to create these variables in the apply function. What do you think?